I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 54, in script
    ksr "the class average was still a 20%"
ValueError: incomplete format

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 54, in script
    ksr "the class average was still a 20%"
  File "C:\Users\anarn\Downloads\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\anarn\Downloads\renpy-8.3.4-sdk\renpy\ast.py", line 621, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\anarn\Downloads\renpy-8.3.4-sdk\renpy\exports\sayexports.py", line 121, in say
    what = what % tag_quoting_dict
ValueError: incomplete format

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.4.24120703
OverDate 1.0
Fri Feb 28 00:46:02 2025
